shared/chat/conversation/info-panel/index.js

```diff
@@ -274,7 +274,7 @@ const typeSizeEstimator = (row: Row): number => {
       return 47
 
     case 'leave channel':
-      return 17
+      return 34
 
```

Comment:
Do we even need these estimates at all?

Reply:
The useful ones are the participant estimates since it will affect how big the scroll container should be. If these are too off then the scroll bar jumps as the size of the list updates.

The early items don't really matter much because you haven't started scrolling yet. So we could safely remove these if we wanted to.

Reply:
Seems like the scroll bar jumping isn't a big deal vs having to keep these values in sync